node-module-system/calculate.ts

```diff
@@ -0,0 +1,17 @@
+import { add } from './add.js';
+import { subtract } from './subtract.js';
+import { multiply } from './multiply.js';
+import { divide } from './divide.js';
+
+const addOperand = add(Number(process.argv[4]), Number(process.argv[2]));
+const minusOperand = subtract(Number(process.argv[4]), Number(process.argv[2]));
+const divideOperand = divide(Number(process.argv[4]), Number(process.argv[2]));
+const multiplyOperand = multiply(
+  Number(process.argv[2]),
+  Number(process.argv[4])
+);
+
+console.log('result:', addOperand);
+console.log('result:', minusOperand);
+console.log('result:', divideOperand);
+console.log('result:', multiplyOperand);
```

Comment:
This is very close, but you should only do the operation that was specified by the second command line argument. So you need to get that argument and switch on it, only performing the operation it specifies.